client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts

```diff
@@ -1,3 +1,8 @@
+
+export const phoneNumberRegex = /^(0(?:[23489]|5[0-689]|7[2346789])(?![01])(\d{7}))$/;
```

Comment:
if there is a use in this regex in more places you can extract it to commons/regex file